<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color: #505050;
font: 15pt &quot;Eras Bold ITC&quot;;;</string>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="username_input">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: white;
	background-color: #7b8087;
	font: 15pt &quot;Eras Bold ITC&quot;;
	border-radius: 2px;
}

QPushButton:pressed{
	padding-top: 5px;
	padding-left: 5px;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color: #505050;
font: 15pt &quot;Eras Bold ITC&quot;;;QPushButton{
	color: white;
	background-color: #7b8087;
	font: 15pt &quot;Eras Bold ITC&quot;;
	border-radius: 2px;
}

QPushButton:pressed{
	padding-top: 5px;
	padding-left: 5px;
}</string>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="password_input">
     <property name="minimumSize">
      <size>
       <width>10</width>
       <height>50</height>
      </size>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="login_button">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: white;
	background-color: #7b8087;
	font: 15pt &quot;Eras Bold ITC&quot;;
	border-radius: 2px;
}

QPushButton:pressed{
	padding-top: 5px;
	padding-left: 5px;
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="register_button">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	color: white;
	background-color: #7b8087;
	font: 15pt &quot;Eras Bold ITC&quot;;
	border-radius: 2px;
}

QPushButton:pressed{
	padding-top: 5px;
	padding-left: 5px;
}</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
